docs/installation.rst

```diff
@@ -724,6 +726,15 @@ Druid
 Note that you can run the ``superset refresh_druid`` command to refresh the
 metadata from your Druid cluster(s)
 
+Dremio
+-----
+
+Install the following dependencies to connect to Dremio:
+
+* pyodbc `pip install pyodbc`
```

Comment:
I don't think this step is needed, as it's already a requirement in `sqlalchemy_dremio`: https://github.com/naren-dremio/sqlalchemy_dremio/blob/master/setup.py#L14-L18